tests/plugins/mocha.test.support/hooks/hook.js

```diff
@@ -32,24 +33,66 @@ exports.init = (logger, config, cli) => {
 
 	cli.on('build.post.compile', async (builder, done) => {
 		if (builder.platformName === 'android') {
-			await wakeDevices(logger, builder).catch(e => logger.warn(`could not wake ${builder.deviceId}: ${e}`));
+			const emulatorManager = new EmulatorManager(config);
+
+			await wakeDevices(logger, builder, emulatorManager)
+				.catch(e => logger.warn(`Could not wake ${builder.deviceId}: ${e}`));
+			await enableAutoRotation(logger, builder, emulatorManager)
+				.catch(e => logger.warn(`Could not enable auto-rotation ${builder.deviceId}: ${e}`));
 		}
 		done();
 	});
 };
 
-async function adb(argumentArray) {
-	return new Promise(resolve => appc.subprocess.run(ADB_PATH, argumentArray, { shell: false, windowsHide: true }, (error, stdout, _stderr) => {
-		resolve(stdout);
-	}));
+async function adb(args) {
+	return new Promise((resolve, reject) => {
+		let child;
+		let stdout = '';
+		let stderr = '';
+
+		if (process.platform === 'win32') {
+			child = spawn(
+				process.env.comspec || 'cmd.exe',
+				[ '/S', '/C', '"', 'adb', ...args,'"' ],
```

Comment:
<!--
  1 failure:  tests/plugins/moc...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: tests/plugins/mocha.test.support/hooks/hook.js;
  Line: 56;
-->
- :no_entry_sign: tests/plugins/mocha.test.support/hooks/hook.js line 56 – A space is required after ','. (comma-spacing)